README.md

```diff
@@ -171,15 +179,27 @@ There’s [`babel-plugin-styled-components`](https://github.com/styled-component
 
 **Migrate to `unfetch` with caution.** While it supports the most popular API parts, your app might break if it relies on something less common.
 
-## Unspecific Libraries
+## reactstrap
+
+### [`babel-plugin-transform-imports`⤵️](#babel-plugin-transform-imports)
+
+## react-bootstrap
+
+### [`babel-plugin-transform-imports`⤵️](#babel-plugin-transform-imports)
+
+## react-router
+
+### [`babel-plugin-transform-imports`⤵️](#babel-plugin-transform-imports)
+
+## Solutions that work with multiple libraries
 
-Of course, there are also optimization tips for other libaries too. You can use them with common sense to get smaller / more performant bundles.
+Of course, there are also optimization tips for other libaries too. You can use them with common sense to get smaller or more performant bundles.
 
 ### `babel-plugin-transform-imports`
```

Comment:
maybe would be good to mention that this wouldn't be much of a problem if we help libraries migrate to flat bundles?

Reply:
Hm, I believe suggesting to contribute is out of the repo’s scope.

I.e., I see the repo as a list of specific optimization instructions. Contribution suggestions are generally useful, but they fall out of that premise.